### ISSUE TITLE
Add ability to use templates in fields like "hours_to_show"

### ISSUE THREAD

**Issue by andyfrei** [royto/logbook-card#105]
**Is your feature request related to a problem? Please describe.**
It would be great if we could use templating in fields like "hours_to_show", because then you could display only for hours that a person has been away for example > eg a great overview what happened when been away

**Describe the solution you'd like**
Add ability to use jinja2 templating for field input like for "hours_to_show"

**Describe alternatives you've considered**
reading from an input number helper to input into field would also be great

**Comment by igorski88** [comment]
Yes!! Please add the ability to use templating. The service call I want should be based off of the state and templating is the only way I think I can achieve this.

**Comment by MichaelSalis** [comment]
I wanted to create a tabs view without having a bunch of logbooks, and I wanted to use a template on my hidden on the  custom_log_map this would be a great addition
